web/modules/custom/do_base/src/Controller/BlogRssFeedController.php

```
@@ -0,0 +1,104 @@
+    $query->fields('n', ['nid']);
+    $query->condition('n.type', 'civictheme_page');
+    $query->condition('n.status', 1);
+    $query->condition('pa.alias', '/blog/%', 'LIKE');
+    $query->condition('pa.alias', '/blog', '<>');
+    $query->orderBy('n.created', 'DESC');
+    $query->range(0, 20);
+    $nids = $query->execute()->fetchCol();
+
+    $nodes = $nids ? $this->entityTypeManager()->getStorage('node')->loadMultiple($nids) : [];
+
+    $site_config = $this->config('system.site');
```

PHPMD (2.15.0)

[error] The variable $site_config is not named in camelCase. (undefined)

(CamelCaseVariableName)